README.md

```diff
@@ -1,4 +1,184 @@
-Astarte Dashboard
-=================
+<!-- PROJECT LOGO -->
+<br />
+<p align="center">
+  <a href="https://github.com/astarte-platform/astarte-dashboard">
+    <img src="src/static/img/logo-login.svg" alt="Logo" height="80">
+  </a>
 
-Astarte Dashboard allows users perform administrative tasks such as installing interfaces and triggers.
+  <h3 align="center">Astarte Dashboard</h3>
+
+  <p align="center">
+    A graphical management tool for Astarte realms
+  </p>
+</p>
+
+
+<!-- TABLE OF CONTENTS -->
+<details open="open">
+  <summary>Table of Contents</summary>
+  <ol>
+    <li><a href="#about-the-project">About The Project</a></li>
+    <li>
+      <a href="#getting-started">Getting Started</a>
+      <ul>
+        <li><a href="#prerequisites">Prerequisites</a></li>
+        <li><a href="#configuration">Configuration</a></li>
+        <li><a href="#starting-up-a-local-server">Starting up a local server</a></li>
+        <li><a href="#testing">Testing</a></li>
+      </ul>
+    </li>
+    <li><a href="#license">License</a></li>
+  </ol>
+</details>
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+[![Dashboard Home Screen Shot][dashboard-home-screenshot]](https://github.com/davidebriani/astarte/blob/docs/add-astarte-dashboard-documentation/doc/images/astarte-dashboard-realm-overview.png)
+
+The Astarte Dashboard is a graphical tool that allows users to analyze the status of
+their realm and perform administrative tasks such as installing interfaces and triggers,
+manage devices and run pipeline flows on your Astarte realm.
+
+
+<!-- GETTING STARTED -->
+## Getting Started
+
+The following steps will help you to get your dashboard up and running locally but
+the same steps can be followed if you need to deploy the dashboard in a remote server.
+
+### Prerequisites
+
+Before starting, you must have an Astarte instance up and running, either locally
+or on a remote cluster. You can have a look at the [Astarte in 5 minutes](https://docs.astarte-platform.org/1.0/010-astarte_in_5_minutes.html#content) if
+you haven't already.
+
+As the Dashboard is build with javascript, you'll also need the following tools:
+* node (10 or greater)
+* npm (6 or greater)
+
+### Configuration
+
+The dashboard relies on a configuration file for parameters like the Astarte API URL.
+As soon as you open the dashboard, it will search for a file `config.json`
+containing the following keys
+
+* __astarte_api_url__ (required)
+  The base URL of your Astarte API endpoints.
+  This will be used to deduct the endpoints of all Astarte components
+
+  + AppEngine: api_url + /appengine
+  + Realm Management: api_url + /realmmanamenet
+  + Pairing: api_url + /pairing
+  + Flow: api_url + /flow
+
+  If the special string `localhost` is set, it will set the Astarte in 5 minutes
```

Comment:
If the special string `localhost` is set, it will set the same defaults used for the *Astarte in 5 minutes* guide: